packages/wallet-sdk/package.json

```diff
@@ -22,7 +22,7 @@
     "pretest": "node compile-assets.js",
     "test": "jest",
     "test:coverage": "yarn test:unit && open coverage/lcov-report/index.html",
-    "prebuild": "rm -rf ./build && node -p \"'export const LIB_VERSION = \\'' + require('./package.json').version + '\\';'\" > src/version.ts",
+    "prebuild": "rm -rf ./dist && node -p \"'export const LIB_VERSION = \\'' + require('./package.json').version + '\\';'\" > src/version.ts",
```

Comment:
`build` directory doesn't exist. Changing this to remove the `dist` directory before building